apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -15,7 +15,8 @@ const TutorialExplorer = React.createClass({
   propTypes: {
     tutorials: React.PropTypes.array.isRequired,
     filterGroups: React.PropTypes.array.isRequired,
-    initialFilters: React.PropTypes.objectOf(React.PropTypes.arrayOf(React.PropTypes.string)).isRequired,
+    initialFilters: React.PropTypes.objectOf(React.PropTypes.arrayOf(React.PropTypes.string)),
+    hideFilters: React.PropTypes.objectOf(React.PropTypes.arrayOf(React.PropTypes.string)),
```

Comment:
@Bjvanminnen out of curiosity, is there a way to use shape to avoid this duplication of filter description?